Cargo.toml

```
@@ -40,12 +41,12 @@ tower-http = { version = "0.4", features = ["trace"] }
 tracing = "0.1"
 web3 = "0.19"
 
-contracts = { git = "https://github.com/cowprotocol/services.git", tag = "v2.253.0", package = "contracts" }
-ethrpc = { git = "https://github.com/cowprotocol/services.git", tag = "v2.253.0", package = "ethrpc" }
-observe = { git = "https://github.com/cowprotocol/services.git", tag = "v2.253.0", package = "observe" }
-shared = { git = "https://github.com/cowprotocol/services.git", tag = "v2.253.0", package = "shared" }
-dto =  { git = "https://github.com/cowprotocol/services.git", tag = "v2.255.1-temp-solvers", package = "solvers-dto" }
-rate-limit =  { git = "https://github.com/cowprotocol/services.git", tag = "v2.253.0", package = "rate-limit" }
+contracts = { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.0", package = "contracts" }
+ethrpc = { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.0", package = "ethrpc" }
+observe = { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.0", package = "observe" }
+shared = { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.0", package = "shared" }
+dto =  { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.1-temp-solvers-dto", package = "solvers-dto" }
```

Do we still need the solvers temp post fix for this version?
```suggestion
dto =  { git = "https://github.com/cowprotocol/services.git", tag = "v2.258.0", package = "solvers-dto" }
```

---

The tag `v2.258.1-temp-solvers-dto` was created by me when I wanted to solve the `deny_unkonwn_fields`. By setting it to `v2.258.0` (latest), we should already include that change directly :ok_hand:

---

> The tag v2.258.1-temp-solvers-dto was created by me

Not really, it was the `v2.255.1-temp-solvers` tag. I created a new tag to publish an image with the latest solver-dto changes that are not part of the `v2.258.0` and `v2.258.1`.